packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -297,6 +303,10 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 		itExpects(
 			"Send ops fails for swept datastores in summarizing container loaded before tombstone timeout",
 			[
+				{
+					eventName: "fluid:telemetry:ContainerRuntime:GC_DeletingLoadedDataStore",
+					clientType: "noninteractive/summarizer", // summarizationWithUnreferencedDataStoreAfterTime has a summarizer spanning before/after the delete
+				},
```

Comment:
I wonder if you'll run into any test flakiness due to logging at different times.

Reply:
We shall see